src/App.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react'
+import JournalistDashboard from './components/JournalistDashboard'
 
 const App = () => {
   return (
     <div>
-      Hello world
+     <JournalistDashboard />
```

Comment:
The name `JournalistDashboard` seems to scope down the Darscboard to just one role. Is that intentional?

Reply:
Yes, we thought that we scope it down because journalist and editor will have quite different dashboards. Also as we plan to use router for login feature it is handy to have 2 dashboards to switch to depending on who is logging in. It seems simpler than to  use ternaries.